src/scores/categorical/contingency_impl.py

```diff
@@ -584,6 +584,9 @@ def make_contingency_manager(
         if not event_threshold:
             event_threshold = self.default_event_threshold
 
+        if not op_fn:
```

Comment:
```suggestion
        if op_fn is None:
```

is more explicit, but maybe it's okay for now?. There's a lot of things in `python` that are true-y or false-y, but None is always None.

Not sure how this is handled across the code-base, so maybe one for a broader look.

Reply:
I never know which way to lean on this one. I'm not aware of anything that's both callable and Falsey. I am a bit more of a duck typer than a type-in-advance-checker, but I'm aware preferences vary. I don't see any practical implications here.

Reply:
I've gone with if not callable. Let me know what you think of the new version.

Reply:
I've changed my mind again to use "if op_fn is None"

Reply:
yea that's fine as well

Reply:
That way, if someone supplies something erroneous like "hello" or 54, it won't go back to the default, but will try to call it and at least you'll get a stack trace.